client/java/src/main/java/io/openlineage/client/dataset/namespaceResolver/DatasetNamespaceResolver.java

```diff
@@ -0,0 +1,23 @@
+/*
+/* Copyright 2018-2024 contributors to the OpenLineage project
+/* SPDX-License-Identifier: Apache-2.0
+*/
+
+package io.openlineage.client.dataset.namespaceResolver;
```

Comment:
Everywhere in this PR where we use dot case: maybe `namespace.resolver` instead of `namespaceResolver`?

I don't like mixing up the case styles - let's go with regular Java `dot.separated.case` in packages/directory names and Spark/Flink conf.